internal/cli/apps.go

```diff
@@ -86,8 +87,8 @@ var CmdAppShow = &cobra.Command{
 	},
 }
 
-// TODO: When pushing the app again after scaling the instances is reset (?)
-// CmdAppShow implements the epinio `apps show` command
+// CmdAppUpdate is used by the epinio `apps show` command to show information
```

Comment:
This should be `apps update` command instead of `apps show` command.

Reply:
And the description is for `show` also, i.e. incorrect for `update`.

Reply:
Ah, I only fixed half of the message...